game_design/areas/navgiari_city/design.txt

```diff
@@ -0,0 +1,21 @@
+Navgiari City is the capital of the Kingdom of Navgiari and its largest settlement.
```

Comment:
Having Navgiari be the name for both an empire and the capital thereof is something that could get quickly confusing. For example, if somebody said "It will spell doom for Navgiari", then it could either be the capital city or the empire, such as how we refer to Washington D.C. as simply 'Washington'.

Reply:
Counter example Mexico City – it's definitely possible to make that work, and it should in most cases be fairly evident from context.